chalicelib/checks/system_checks.py

```diff
@@ -40,22 +40,24 @@ def wipe_build_indices(connection, es_url):
     client = es_utils.create_es_client(es_url, True)
     full_output = []
     _, indices = cat_indices(client)  # index name is index 2 in row
-    for index in indices:
-        try:
-            index_name = index[2]
-        except IndexError:  # empty [] sometimes returned by API call
-            continue
-        if re.match(BUILD_INDICES_REGEX, index_name) is not None:
-            try:
-                resp = Retry.retrying(client.indices.delete, retries_allowed=3)(index=index_name)
-            except Exception as e:
-                full_output.append({'acknowledged': True, 'error': str(e)})
-            else:
-                full_output.append(resp)
-
-    if any(output['acknowledged'] is not True for output in full_output):
-        check.status = 'FAIL'
-        check.summary = check.description = 'Failed to wipe all test indices, see full output'
+    # XXX: Commented out so we can see if the below logic is what is causing timeout on
```

Comment:
So the core of the work is not being done here? Should it at least put out a message saying so?